scene/gui/control.cpp

```diff
@@ -197,6 +197,12 @@ void Control::reparent(Node *p_parent, bool p_keep_global_transform) {
 
 // Editor integration.
 
+int Control::root_dir = 0;
```

Comment:
I'd prefer if we used a full name so it's not confused with anything else, like directory.